source/MaterialXGenShader/Util.cpp

```diff
@@ -75,12 +75,12 @@ void loadDocuments(const FilePath& rootPath, const FileSearchPath& searchPath, c
     }
 }
 
-void loadLibrary(const FilePath& file, DocumentPtr doc)
+void loadLibrary(const FilePath& file, DocumentPtr doc, const FileSearchPath* searchPath)
 {
     DocumentPtr libDoc = createDocument();
     XmlReadOptions readOptions;
     readOptions.skipConflictingElements = true;
-    readFromXmlFile(libDoc, file, FileSearchPath(), &readOptions);
+    readFromXmlFile(libDoc, file, searchPath ? *searchPath : FileSearchPath(), &readOptions);
```

Comment:
Not yet included here but stdlib_ng.mtlx should include stdlib_defs.mtlx, which will fail without a search path. (stdlib_ng.mtlx is a relative path).